config/locales/en.yml

```diff
@@ -35,8 +35,9 @@ en:
 
   labels: &LABELS
     assigned_to: Assigned to
+    application_information: Application information
+    search_criteria: Search criteria
     applicant_date_of_birth: Applicant's date of birth
-    all_caseworkers: Assigned to
```

Comment:
This was unused as far as I could see

Reply:
Yep I think its not, probably just a leftover.